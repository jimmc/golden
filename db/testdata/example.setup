CREATE table test(n int, s string);
INSERT into test(n, s) values(1, 'a'), (2, 'b'), (3, 'c'), (4, 'd');
