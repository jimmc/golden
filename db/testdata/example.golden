s="a", n=1
s="b", n=2
s="c", n=3
s="d", n=4
